fr-FR/lang/numere.nlng

```diff
@@ -0,0 +1,1186 @@
+PARSERFUNCS_LISTFUNC_METHOD_ZSCOREOF_[DATA]=				zscoreof(nCol,nRows={1:nrows})                             {VAL}     - Calcule la standardisation des valeurs par rapport à leur écart par rapport à la moyenne (également appelé z-score) dans la colonne nCol pour les lignes nRows. Les lignes peuvent être prédéfinies et correspondent par défaut à la dimension complète. La moyenne et l'écart-type sont toujours calculés par rapport aux lignes sélectionnées
+# Kommandotabelle
+PARSERFUNCS_LISTCMD_HEADLINE=Kommandosatz
+PARSERFUNCS_LISTCMD_TYPE_EVAL=Auswerten
+PARSERFUNCS_LISTCMD_TYPE_PLOTTING=Plotten
+PARSERFUNCS_LISTCMD_TYPE_FLOWCTRL=Kontrollfluss
+PARSERFUNCS_LISTCMD_TYPE_PROC=Prozeduren
+PARSERFUNCS_LISTCMD_TYPE_DATACREATION=Datenerzeugung
+PARSERFUNCS_LISTCMD_TYPE_LOADSAVE=Laden und Speichern
+PARSERFUNCS_LISTCMD_TYPE_DEF=Definitionen
+PARSERFUNCS_LISTCMD_TYPE_DATACTRL=Datenkontrolle
+PARSERFUNCS_LISTCMD_TYPE_MAINCTRL=Steuerung
+PARSERFUNCS_LISTCMD_TYPE_MISC=Verschiedenes
+PARSERFUNCS_LISTCMD_TYPE_GUI=Graphische Oberfläche
+PARSERFUNCS_LISTCMD_TYPE_PLUGINS=Plugins
+PARSERFUNCS_LISTCMD_TABLEHEAD=							Befehl:      Syntax und Ergebnis:       Fonction :
+PARSERFUNCS_LISTCMD_CMD_ABORT_[FLOWCTRL]=				abort                                   - Bricht eine interaktive Schleifendefinition oder Plotkomposition in der Konsole ab, so dass die Konsole wieder normal verwendet werden kann
+PARSERFUNCS_LISTCMD_CMD_ABOUT_[MAINCTRL]=				about                                   - Zeigt Lizenz- und Urheberinformationen zu NumeRe in der Konsole an
+PARSERFUNCS_LISTCMD_CMD_APPEND_[LOADSAVE]=				append       NM [-PAR]  -> {VAL}        - Attache les données du fichier NM aux données déjà existantes en mémoire. Ce paramètre permet d'attacher plusieurs fichiers à la fois. Renvoie les index des données jointes dans le tableau cible
+PARSERFUNCS_LISTCMD_CMD_ASSERT_[MAINCTRL]=				assert       EX  -> {ARG}               - Vérifie la véracité de l'expression passée en paramètre et la renvoie. Si elle n'est pas vraie, la commande renvoie une erreur. Cette commande est destinée à être utilisée dans un test de logiciel
+PARSERFUNCS_LISTCMD_CMD_AUDIOREAD_[LOADSAVE]=			audioread    NM [-set PAR]  -> {VAL}    - Lit les métadonnées d'un fichier WAVE ou le fichier entier si une table cible a été spécifiée
+PARSERFUNCS_LISTCMD_CMD_AUDIO_[LOADSAVE]=				audio        OB [-PAR]                  - Crée un fichier WAVE à partir de l'ensemble de données dans OB. Le taux d'échantillonnage et le nom du fichier cible peuvent être définis à l'aide de paramètres
+PARSERFUNCS_LISTCMD_CMD_BITMAP_[GUI]=					bitmap       EX [-set PAR]              - Insère une image dont le nom de fichier a été spécifié via EX dans un layout de fenêtre
+PARSERFUNCS_LISTCMD_CMD_BREAK_[FLOWCTRL]=				break                                   - Bricht die aktuelle Schleife ab oder verlässt den aktuellen case eines switch-Blocks und springt im Programmfluss eine Ebene höher
```

Comment:
IT can not be automatically detected, because there is no "Sytanx und Ergebnis" --> they should be done manually.
Another possibility is to always have a file where we save these kind of lines to automatically translate them

Reply:
If you referring talking about how to detect the start of the actual documentation in such scenarios: you can use the tags itself to determine the needed syntax, i.e. tags starting with `PARSERFUNCS_LISTCMD_CMD_` will always have a ` - ` (whitespace, minus, whitespace) section after which the documentation starts. Similar rules can be obtained for other tag groups, i.e. functions and methods.